### PR TITLE
Make themeable sass variables customizable + update colors to match master

### PR DESCRIPTION
**Pull request by kscherling**
This pr ensures that sass variables can be redefined by consuming apps by adding the '!default' flag.  Simultaneously, it updates grays and other color variables to reflect the changes made to master.

### QA Steps

- Test in concert with [#161](https://invent.focusvision.com/Portland/revelation-frontend-admin/pull/161)

### CODE REVIEW

File: addon/pods/components/rui-dropdown-trigger/component.js [hunk 1]
```diff
@@ -6,8 +6,13 @@ export default Ember.Component.extend({
   tagName: 'button',
   classNames: ['dropdown-toggle'],
   classNameBindings: ['computedStyle'],
-  attributeBindings: ['type', 'data-toggle', 'haspopup', 'aria-expanded'],
-  id: '',
+  attributeBindings: [
+    'type',
+    'data-toggle',
+    'haspopup',
```
Comment (smashedlife) [inline addon/pods/components/rui-dropdown-trigger/component.js]: Shouldn't this be `data-has-popup`?

Comment (kscherling) [inline addon/pods/components/rui-dropdown-trigger/component.js]: no

Comment (smashedlife) [inline addon/pods/components/rui-dropdown-trigger/component.js]: Why is that?

Comment (kscherling) [inline addon/pods/components/rui-dropdown-trigger/component.js]: Looks like it likely should be aria-haspopup actually.

Comment (kscherling) [inline addon/pods/components/rui-dropdown-trigger/component.js]: Likely from here.  Not sure why the aria part didn't make it in.
http://v4-alpha.getbootstrap.com/components/dropdowns/#example

**Comment by kscherling** [comment]
@smashedlife This should be fixed

**Pull request merged**